### ISSUE TITLE
[Question]: a question about randomtrips and randomSample

### ISSUE THREAD

**Issue by zhangm269** [eclipse/sumo#13338]
- I have a question: after using randomtrips to generate trips, I found that there are too many right-turning vehicles, which doesn't quite fit my scenario setup.
- Now I want to set the ratio of left-turning straight and right-turning vehicles at each intersection to 30% 60% 10%. Is there a python command for this? Is it possible to set the turn flow rate via netedit? 

**SUMO-version:**
1.17.0
**operating system:**
Windows

**Comment by namdre** [comment]
There is currently some tooling to generate turn ratio files but nothing that allows setting default ratios.
see https://sumo.dlr.de/docs/Demand/Routing_by_Turn_Probabilities.html#automatic_generation_of_turn_definitions
You can also create/edit turn turn count and turn ratio files with netedit using the 'data mode'

**Comment by namdre** [comment]
#9995 proposes to allow setting default turning ratios directly in routeSampler.py but it isn't clear whether the resulting amount of constraints is feasible for optimization (it shouldn't present a problem for sampling).  Having a stand-alone python tool to generate the files allows more control (i.e. to only set ratios for key intersections).  The existing tool generateTurnDefs.py doesn't load a .net.xml file (only a .con.xml file) and thus cannot make use of computed turning directions.  A new tool would probably make generateTurnDefs.py obsolete.

**Comment by Daedalus-Icarus** [comment]
@behrisch @namdre can I work on this issue as first contribution?

**Comment by namdre** [comment]
> [@behrisch](https://github.com/behrisch) [@namdre](https://github.com/namdre) can I work on this issue as first contribution?

Yes. Please take notice of https://sumo.dlr.de/docs/FAQ.html#how_do_code_contributions_work
